### PR TITLE
Add racket-langserver support

### PR DESCRIPTION
**Pull request by Lazerbeak12345**

### CODE REVIEW

**Comment by Lazerbeak12345** [comment]
I've no idea why the mac test failed, but it looks like something with brew... Not something I would know what to do about.

**Comment by Lazerbeak12345** [comment]
Also, is calling it "racket" the best idea? I can change it to `racketls`, `racket-language-server` or something else like that, as this does use [racket-language-server](https://github.com/theia-ide/racket-language-server)

File: lua/nvim_lsp/racket.lua [hunk 1]
```diff
@@ -0,0 +1,23 @@
+local configs = require 'nvim_lsp/configs'
+local util = require 'nvim_lsp/util'
+
+configs.racket = {
+  default_config = {
+    cmd = {"racket", "-l", "racket-langserver"};
+    filetypes = {"racket"};
```
Comment (ckipp01) [inline lua/nvim_lsp/racket.lua:7]: Btw, this should probably be `scheme`. Be default `.rkt` files will register as that.

Comment (mjlbach) [reply lua/nvim_lsp/racket.lua:7]: You can have both

**Comment by mjlbach** [comment]
@Lazerbeak12345 Please see my notes on https://github.com/neovim/nvim-lspconfig/pull/679 which is a duplicate

**Comment by mjlbach** [comment]
Also, as amusing as they are, please squash your commits :P

**Comment by mjlbach** [comment]
I do also prefer racket_langserver

**Comment by Lazerbeak12345** [comment]
Thanks for the feedback! I’ll look into those things when I have time.

**Comment by Lazerbeak12345** [comment]
I took a look, and I do think @ckipp01 's pr is better.

**Pull request closed without merging**